src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -145,6 +153,16 @@ object SbtLicenseReport extends AutoPlugin {
     licenseFilter := TypeFunctions.const(true),
     licenseReportStyleRules := None,
     licenseReportTypes := Seq(MarkDown, Html, Csv),
-    licenseReportColumns := Seq(Column.Category, Column.License, Column.Dependency)
+    licenseReportColumns := Seq(Column.Category, Column.License, Column.Dependency),
+    licenseCheckAllow := Seq(
+      LicenseCategory.Apache,
+      LicenseCategory.BouncyCastle,
+      LicenseCategory.BSD,
+      LicenseCategory.CC0,
+      LicenseCategory.EPL,
+      LicenseCategory.MIT,
+      LicenseCategory.Mozilla,
+      LicenseCategory.PublicDomain
+    )
```

Comment:
Why was the `LGPL` category not added to this list? AFAIK, it is allowed to ship libraries licensed under the LGPL with proprietary software.

Reply:
?